build.sbt

```diff
@@ -26,7 +31,9 @@ def commonSettings: Seq[Setting[_]] = Seq(
   testOptions += Tests.Argument(TestFrameworks.ScalaCheck, "-w", "1"),
   javacOptions in compile ++= Seq("-target", "6", "-source", "6", "-Xlint", "-Xlint:-serial"),
   incOptions := incOptions.value.withNameHashing(true),
-  crossScalaVersions := Seq(scala210)
+  crossScalaVersions := Seq(scala210),
+  bintrayPackage := (bintrayPackage in ThisBuild).value,
+  bintrayRepository := (bintrayRepository in ThisBuild).value
```

Comment:
It is rewiring what bintray-sbt adds to each project.
It's not a noop since `commonSettings` gets appended to all subprojects as a project-level setting whereas `(bintrayRepository in ThisBuild).value` is a build-level setting.